system-tests/resources/cli-tools/entrypoint.sh

```diff
@@ -28,36 +28,29 @@ function seedVerifiedCredentials() {
      java -jar identity-hub-cli.jar \
                  -s="http://$participantName:8181/api/identity-hub" \
                  vc add \
-                 -c='{"id":"'$vcId'","credentialSubject":{'"$subject"'}}' \
+                 -c='{"id":"'"$vcId"'","credentialSubject":{'"$subject"'}}' \
```

Comment:
Why are the outermost quotes required?

Reply:
It's an IDE warning otherwise, and good practice in general to avoid splitting on spaces

Reply:
I think you are referring to the innermost double quotes, which I agree. But what about the outermost double quotes, are they needed?

Reply:
One set of quotes (inner) are for the shell, and have no effect (if the variable does not contain spaces). One set of quotes (outer) are verbatim in JSON result.